src/components/TodoItem.tsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import { Todo } from '../types/Todo';
+import cn from 'classnames';
+import classNames from 'classnames';
+
+type Props = {
+  todo: Todo;
+  isLoading?: boolean;
+  onDeleteTodos: (todoId: number) => Promise<void>;
+};
+
+export const TodoItem: React.FC<Props> = props => {
+  const { todo, isLoading, onDeleteTodos } = props;
+
+  return (
+    <div
+      key={todo.id}
+      data-cy="Todo"
+      className={cn('todo', { completed: todo.completed })}
+    >
+      <label className="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+          checked={todo.completed}
+          onChange={() => {}}
+        />
+      </label>
+
+      <span data-cy="TodoTitle" className="todo__title">
+        {todo.title}
+      </span>
+
+      <button
+        type="button"
+        className="todo__remove"
+        data-cy="TodoDelete"
+        onClick={() => onDeleteTodos(todo.id)}
+        disabled={isLoading}
+      >
+        ×
+      </button>
+
+      {isLoading && (
```

Comment:
This is an unnecessary condition - you activate this loader on 49 line

Reply:
Good point